vNext/samples/ODataSample.Web/Models/ISampleService.cs

```diff
@@ -9,5 +7,7 @@ public interface ISampleService
     {
         IEnumerable<Product> Products { get; }
         IEnumerable<Customer> Customers { get; }
+        [ODataFunction]
```

Comment:
I didn't find anything up to now. I'm considering adding new types of attributes so we can establish the nature of the operation: ODataBoundFunction, ODataUnboundOperation, ODataBoundAction and ODataUnboundAction. Does that make sense to you?